Tools/TestWebKitAPI/Tests/WebKitCocoa/WritingTools.mm

```diff
@@ -2477,6 +2560,7 @@ InstanceMethodSwizzler swizzler(PAL::getWTWritingToolsClass(), @selector(schedul
     TestWebKitAPI::Util::run(&finished);
 }
 
+#if PLATFORM(IOS_FAMILY)
```

Comment:
Separate change... especially important if the actual change here needs to be reverted.